src/Hedgehog/Property.fs

```diff
@@ -6,6 +6,25 @@ open System
 type Property<'a> =
     | Property of Gen<Journal * Outcome<'a>>
 
+
+type PropertyConfig = internal {
+    TestCount   : int<tests>
+    ShrinkLimit : int<shrinks> option
+}
+
+
+module PropertyConfig =
+    let defaultConfig =
+        {   TestCount = 100<tests>
+            ShrinkLimit = None }
+
+    let withShrinkLimit shrinkLimit c =
+        { c with ShrinkLimit = Some shrinkLimit }
+
+    let withTestCount testCount c =
+        { c with TestCount = testCount }
```

Comment:
These functions need type annotations to be in line with the rest of the code. While we're making new functions, I'd prefer `config` over `c`.

Reply:
Done. Also added some docs.